webdriver/tests/find_element/META.yml

```diff
@@ -0,0 +1,5 @@
+links:
+    - product: chrome
+      url: https://crbug.com/974254
```

Comment:
The test was used as an example in this bug, but the bug seems unrelated to the test failure.